integrations/zendesk/src/events/ticket-assigned.ts

```diff
@@ -8,14 +8,27 @@ export const executeTicketAssigned = async ({
   zendeskTrigger: TriggerPayload
   client: bp.Client
 }) => {
+  const { ticketId, agent } = zendeskTrigger
```

Comment:
HITLINT change number 6) create an event that is hitlAssigned (must be this name) (from webhook handler)